Test/cli/diagnosticsFormats.dfy.expect

```diff
@@ -0,0 +1,24 @@
+diagnosticsFormats.dfy(8,0): Warning: module-level const declarations are always non-instance, so the 'static' keyword is not allowed here
+diagnosticsFormats.dfy(9,17): Error: result of operation might violate newtype constraint for 'byte'
+diagnosticsFormats.dfy(12,16): Error: A precondition for this call might not hold.
+diagnosticsFormats.dfy(11,35): Related location: This is the precondition that might not hold.
+
+Dafny program verifier finished with 1 verified, 2 errors
+{"range":{"filename":"diagnosticsFormats.dfy","range":{"start":{"line":8,"character":0}}},"severity":2,"message":"module-level const declarations are always non-instance, so the \u0027static\u0027 keyword is not allowed here","source":"Parser","relatedInformation":[]}
```

Comment:
To test for the "end", you have to provide the argument name `/showSnippets:1`

Reply:
I don't understand: `/showSnippets` is for showing the error in context, and we don't want to do that in JSON mode, right?

Reply:
`/showSnippets` enables Dafny to create RangeTokens instead of just Tokens.
For the JSON mode, if `/showSnippets` is activated, it would be reasonable to me that we don't add the snippet as part of the error message (but we could... that could be useful), but that way we can report error using ranges.

Reply:
Well the only reason was that I'm too lazy (or don't think it's worth it?) to change the 1200+ files of our integration tests  And it could be a breaking change (however, I don't think this would be a big deal)

Reply:
Oh, good point regarding the test files.

Reply:
I've added `-showSnippets:1` to the test and documented it in UserGuide.md